lib/fluent/plugin/out_elasticsearch.rb

```diff
@@ -27,6 +27,7 @@ class ConnectionFailure < StandardError; end
   config_param :request_timeout, :time, :default => 5
   config_param :reload_connections, :bool, :default => true
   config_param :reload_on_failure, :bool, :default => false
+  config_param :resurrect_after, :time, :default => 5
```

Comment:
can you set default to 60, since it's current default - https://github.com/elastic/elasticsearch-ruby/blob/master/elasticsearch-transport/lib/elasticsearch/transport/transport/base.rb#L11